components/org.wso2.carbon.identity.api.user.authorized.apps/org.wso2.carbon.identity.rest.api.user.authorized.apps.v2/src/main/resources/authorizedApps.yaml

```diff
@@ -233,6 +233,28 @@ paths:
         '500':
           $ref: '#/components/responses/ServerError'
 
+  '/applications/{application-id}/tokens':
```

Comment:
can we make this `authorized-apps` instead of `applications` so that it is consistent with other places?

Reply:
Since the existing admin endpoints in the authorized-app API revokes the tokens for a given authorized user.
In order to maintain restfulness of the API, we have taken this approach as this API revokes all tokens for a given application.